1-ECommerce/src/components/Error.tsx

```diff
@@ -3,7 +3,7 @@ interface ErrorProps {
   subheading: string;
 }
 
-export const Error = ({ heading, subheading }: ErrorProps) => (
+export const ErrorComponent = ({ heading, subheading }: ErrorProps) => (
```

Comment:
Were you having trouble with the `Error` name? Adding `Component` to it seems redundant

Reply:
Yea, it was a suggestion from the linter because `Error` is also a class that can be used for throwing generic errors (`throw new Error('message here')`). Another option would be to keep the name as `Error` and `import ... as` some other name, but if I'm doing this on every import it's better to just change the name. But yea, in any case, the file should have the same name as the component